modules/transactions.js

```diff
@@ -103,8 +103,9 @@ __private.list = function(filter, cb) {
 	const params = {};
 	const where = [];
 
+	// Removing null characters from data filter to avoid errors in pg-promise library
 	if (filter.data) {
-		params.additionalData = Buffer.from(filter.data, 'utf8');
+		filter.data = filter.data.replace(/\0/g, '');
```

Comment:
Now, when we don't convert the field to text in the query, can we pass `Buffer` instance here?